### PR TITLE
Fix @vscode-sqlite3 error message

### PR DESCRIPTION
**Pull request by OlivierCavadenti**
- Fix the error message when sqlite3 driver don't exist.
- Relates #5002
- If I remove my vscode/sqlite3 install, I have the error. If I reinstall it via "npm install @vscode/sqlite3 --save" it's ok for me. I miss something ?

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -189,7 +189,11 @@ class Client extends EventEmitter {
     try {
       this.driver = this._driver();
     } catch (e) {
-      const message = `Knex: run\n$ npm install ${this.driverName} --save`;
+      let driverName = this.driverName;
+      if (driverName === 'sqlite3') {
```
Comment (kibertoad) [inline lib/client.js:193]: I would suggest more generic solution of driverName override map or, even better, make Client instances aware of their driverName

Comment (OlivierCavadenti) [reply lib/client.js:193]: I introduce an alias driver name in sqlite3 index.js, I can't change the driver name by the way (is used by all tests that use testSql...).
By the way, I retest to delete the package, and run the command "npm install @vscode/sqlite3 --save" and I don't have any problem. I can't see more to do here to fix the problem in the related issue.

**Review by kibertoad — APPROVED** [review]

File: lib/client.js (hunk 1 above)
Comment (kibertoad) [reply lib/client.js]: I think current solution is perfect

**Comment by tompsherman** [comment]
I've just spent a day or so chasing this bug in my code... and this thread as well as [Knex Throwing SQLITE Error](https://github.com/knex/knex/issues/5002) helped me fix it. I found two solutions that seem to work:

either the error message needs to be changed to `npm install @vscode/sqlite3 --save` because this fixes the issue but currently the error message reads `npm install sqlite3 --save` which does not fix the issue no matter how many times you install sqlite3
--OR-- 
the driver on line 32  `knex/lib/dialects/sqlite3/index` needs to be changed from `return require('@vscode/sqlite3');` to `return require('sqlite3');`

**Comment by daniellockyer** [comment]
@OlivierCavadenti/@kibertoad: Any indication when this will be released? 🙂 The error message is tripping people up when `@vscode/sqlite3` fails to compile

**Comment by kibertoad** [comment]
Released in 1.0.5

**Comment by daniellockyer** [comment]
Thank you!

**Comment by thefonso** [comment]
I have a small crazy question....Why is Knex dependent on @vscode/sqlite3 ?

I'm currently working a postgres project....no sqlite on my system...at all. Sooo. 

Why is Knex dependent on @vscode/sqlite3 ?

**Comment by kibertoad** [comment]
@thefonso Do you mean migration creation or something else?

**Comment by mrondin1** [comment]
Similar to @thefonso comment above, I'm getting the @vscode/sqlite3 error when I'm creating migrations on an exclusively postgres project. Is that expected?

**Comment by kibertoad** [comment]
No. See #5106
1.0.6 should help in some cases, #5106 should help with the rest.

**Pull request merged**